<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtNetworkClientClass</class>
 <widget class="QMainWindow" name="QtNetworkClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtNetworkClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="m_lineedit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btConnect">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QListWidget" name="m_lwMessages">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>601</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="m_ClickImage">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome To Communication System</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="m_useregitis"/>
    <addaction name="m_userlogin"/>
   </widget>
   <widget class="QMenu" name="menuFind">
    <property name="title">
     <string>Find</string>
    </property>
    <addaction name="m_menuFind"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="m_menuGet_Help"/>
   </widget>
   <addaction name="menuUser"/>
   <addaction name="menuFind"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="m_useregitis">
   <property name="text">
    <string>UserRegitis</string>
   </property>
  </action>
  <action name="m_userlogin">
   <property name="text">
    <string>UserLogin</string>
   </property>
  </action>
  <action name="m_menuGet_Help">
   <property name="text">
    <string>Get Help</string>
   </property>
  </action>
  <action name="m_menuFind">
   <property name="text">
    <string>Find...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtnetworkclient.qrc"/>
 </resources>
 <connections/>
</ui>
